crates/extensions/tedge_config_manager/src/tests.rs

```diff
@@ -63,6 +63,87 @@ fn prepare() -> Result<TempTedgeDir, anyhow::Error> {
         ]"#
         ));
 
+    let plugin_dir = tempdir.dir("config-plugins");
+
+    // Create a mock `file` plugin script
+    let plugin_script = format!(
+        r#"#!/bin/bash
```

Comment:
When you have a large inline content like this shell script, an option is to use the [`include!`](https://doc.rust-lang.org/std/macro.include.html) macro.

Reply:
Resolved by 564077f